postcss-selector-parser.d.ts

```diff
@@ -166,7 +166,7 @@ declare namespace parser {
         replaceWith(...nodes: Node[]): Node;
         next(): Node;
         prev(): Node;
-        clone(opts?: {[override: string]:any}): Node;
+        clone(opts?: {[override: string]:any}): this;
```

Comment:
After cloning the return the value is always an instance of `this`.

With `Node` as the return type you always need to do more type checks after cloning because the specific type was erased.